src/settings-list.js

```diff
@@ -429,5 +429,11 @@ module.exports = [
                 chat.helpers.serverMessage("Chat scrollback is now set to: " + lines, true);
             }
         }
+    },
+    {
+        name: 'Right click user to chat',
```

Comment:
A better name is needed, and it should be formatted properly (capitalized) and inserted into the list properly (alphabetical, ascending).